### DISCUSSION TITLE
Suport for text font editing

### DISCUSSION THREAD

**Discussion by recursively** [illacloud/illa-builder#1199] [Ideas]
I found the ability to edit font is somewhat deficient. Or I just didn't find the right way to adjust the font? I suggest incorporating additional support for text font editing, such as adjusting font weight, font bold, and even adding shadows, etc. Adjusting only the font size is insufficient to meet the requirements. I suggest prioritizing this item ASAP.

**Comment by wenzhuo2022** [comment]
Hello, thanks for your feedback. We will upgrade the style configuration of the component in the near future, which also includes more font editing.
